### PR TITLE
feat: handle toasts

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Continues on https://github.com/cowprotocol/cowswap/pull/4022

This PR handles TOAST events, and shows a snackbar. So now, CoW Swap will not show the toast, and instead the configurator will show the toast below, as it can be seen in the screenshot.

image

# To Test

1. Setup "Dapp mode" for the toasts

image

2. Check that the toast is shown when you do a trade

3. Setup "Self contained" mode for the toasts

3. Check now toasts show in the Dapp only (no snackbars)

### CODE REVIEW

**Comment by elena-zh** [comment]
When I open the PR link to the app (not to the configurator), I don't see connect button at the top, just a line instead 
image

**Comment by elena-zh** [comment]
Explanation toasts are too slow: they are kept to be displayed when I switch the mode, and when I start closing them, I need closing a lot. See the video: https://www.loom.com/share/27fe1b8945e04c718c6020f70fd20141?sid=185a65ee-8f27-4ce5-ae3a-62f8a79cfb87

**Review by elena-zh — CHANGES_REQUESTED** [review]
And I don't see snackbars when I place orders in the Dapp mode (neither for onchain nor of offhchain TXs)

Besides, when I switch to Self-contain mode, I see all previous notifications for TXs that were done in Dapp mode: 
all done before

**Comment by anxolin** [comment]
I'm sorry Elena, there was some changes that I didn't push. So it was not yet connected to the notification logic. I just pushed it. I hope now can be tested

File: apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function useWidgetParams(configuratorState: ConfiguratorState): CowSwapWi
       height: '640px',
       chainId,
       tokenLists: tokenLists.filter((list) => list.enabled).map((list) => list.url),
-      baseUrl: DEFAULT_BASE_URL,
+      baseUrl: 'https://swap-dev-git-feat-snackbars-from-events-8-cowswap.vercel.app', // TODO: Leave default DEFAULT_BASE_URL,
```
Comment (shoom3301) [inline apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts:54]: TODO: revert this after QA

**Review by shoom3301 — APPROVED** [review]

**Comment by shoom3301** [comment]
@elena-zh 
1. Fixed. This PR is looking into https://swap-dev-git-feat-snackbars-from-events-8-cowswap.vercel.app which was not up to date.
2. Fixed
3. This is expected, should be done in another PR (after [this one](https://github.com/cowprotocol/cowswap/pull/3934))
4. This is also expected, because the widget preview don't have changes from this PR. It will be solved once we merge everything in one branch
5. For me it's ok

**Review by elena-zh — APPROVED** [review]
Hey @shoom3301 , thanks for the fixes!

@anxolin , here is one question for you: when notifications go one after another, they are hidden under currently displayed one. WDYT about showing snackbars above/at the bottom of the currently displayed one?

**Pull request merged**